src/db/chromeSotrage.js

```diff
@@ -0,0 +1,80 @@
+/*global chrome*/
+export const ChromeStorage =  {
+     getAllData: () => {
+        return new Promise((resolve, reject) => {
+                chrome.storage.sync.get(null, (data)=>{ resolve(data)});
+        });
+    },
+
+    saveData: (key, value) => {
+        chrome.storage.sync.set({[`${key}`]: value}, ()=>{});
```

Comment:
I think this needs to provide 3rd param which is the callback function, so the caller can have custom callback function for its own usecase. 

Probably something like this?
```javascript
saveData: (key, value, callback) => {
        chrome.storage.sync.set({[`${key}`]: value}, callback);
}
````

Reply:
Fixed!